Claudia/src/main/java/com/endava/task/second/User.java

```diff
@@ -0,0 +1,47 @@
+package com.endava.task.second;
+
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class User {
+    public long id;
+    public String username;
+    public Map<String, Address> addressMap;
+
+    public User(String username, String addressType, Address address) {
+        this.id = Math.abs(new Random().nextLong());
+        this.username = username;
+        this.addressMap = addAddress(addressType, address);
+
+    }
+
+    public Map<String, Address> addAddress(String type, Address address) {
+        Map<String, Address> newAddress = new HashMap<>();
+        newAddress.put(type, address);
+        return newAddress;
```

Comment:
This method wouldn't add an address to new created user. What is the logic here?

Reply:
You're right. Changed.